tests/shared/command.py

```diff
@@ -46,6 +46,7 @@ def kill_process_tree(self, pid):
             print(err)
 
     def run(self, timeout=3600, assert_returncode_zero=True):
+        print(f"Running command: {self.cmd}")
```

Comment:
Look like change for debug.

Reply:
No, actually this is so that the tests that launch a python script in a subprocess would always dump the command itself into log - the developer can then copy-and-paste it and launch it directly when tracing a bug instead of having to launch the test in a debugger and `print`ing the `self.cmd` variable just to learn what command line was launched.